test/morphometrics/test_compute_morphometrics.py

```diff
@@ -170,6 +253,32 @@ def test_get_axon_morphometrics_with_unexpected_myelin_mask_simulated_axons(self
             assert axon_prop['myelin_thickness'] == pytest.approx(0.0, rel=0.01)
             assert axon_prop['myelin_area'] == pytest.approx(0.0, rel=0.01)
             assert axon_prop['gratio'] == pytest.approx(1.0, rel=0.01)
+    
+    @pytest.mark.unit
+    def test_get_axon_morphometrics_with_unexpected_myelin_mask_simulated_axons_with_axon_as_ellipse(self):
```

Comment:
Since all the test images generated for this test are perfect circles, I feel like all these tests dealing with the the ellipse settings are missing some of the functionality. We should have a test image with some ellipses with known minor and major axes, and test that your functionality returns these parameters. i.e. I know all your tests pass, but they may not be testing the primary functionality of this new feature, so it's difficult for me as a new user read the tests and understand the expected behaviour/functionality.

Reply:
That's a good catch. I've made a test image containing axons which are ellipse. 

![Untitled Diagram (4)](https://user-images.githubusercontent.com/31419210/101942449-8a748480-3c0f-11eb-9b7e-e5bb5ddf200c.png)

Let me know what do you think about this image. In the tests, I'm planning to add this test image for computing the diameter using `ellipse_minor_axis`. For this image, I'll do manual measurements on them  which should be similar to the new functionality.

Reply:
there are black pixels between white and gray (shouldn't be the case)

Reply:
Right; corrected the image.